docs/reference/signals.rst

```diff
@@ -127,6 +127,24 @@ The best way to distinguish between a 'move' and 'reorder' is to compare the ``u
     # Register a receiver
     pre_page_move.connect(clear_old_page_urls_from_cache)
 
+``page_url_path_changed``
+-------------------------
+
+This signal is emitted from a ``Page`` after its ``url_path`` value has changed as a result of:
+
+- The publishing of a change to it's `slug`.
```

Comment:
What about publishing a change to an ancestors slug?

Reply:
The `url_path` updates to descendants happens a single update() - I wouldn't want to jeopardise the efficiency of that by emitting the signal for every descendant.

Reply:
Essentially, the signal is only emitted for the page the change was made directly to.

Reply:
That makes sense, Thanks!